pipelines/coaddQAPlots.yaml

```diff
@@ -0,0 +1,234 @@
+description: Make coadd plots for QA
```

Comment:
My main request for this lovely new yaml file is more documentation. Please explain what each plot is doing and pretend I have no idea (spoiler, I don't!).